src/utils/translations/pt.ts

```diff
@@ -0,0 +1,94 @@
+const pt = {
+  // Home Tab
+  Home: "Início",
+  "Period in": "Período em",
+  "no info": "desconhecido",
+  Period: "Período",
+  today: "hoje",
+  Delay: "Atraso",
+  Days: "(0)[Dias];(1)[Dia];(2-4)[Dias];(5-20)[Dias];(21)[Dia];(22-24)[Dias];(25-30)[Dias];(31)[Dia];(32-34)[Dias];(35-40)[Dias];",
+  "Current cycle day": "Dia atual do ciclo",
+  day: "{{count}}º dia",
+  Ovulation: "Ovulação",
+  possible: "pode ser hoje",
+  finished: "terminado",
+  tomorrow: "amanhã",
+  in: "em",
+  "chance of getting pregnant": "probabilidade de engravidar",
```

Comment:
I think `probabilidade` here means a more specific probability in the form of a number